packages/client/src/components/interface/Navigation.tsx

```diff
@@ -371,380 +238,43 @@ const NavigationView = (props: IFullProps) => {
-                <NavigationItem
-                  icon={() => <Icon name="Activity" size="medium" />}
-                  label={intl.formatMessage(navigationMessages['statistics'])}
-                  onClick={goToPerformanceStatistics}
-                  id="navigation_statistics"
-                  isSelected={
-                    enableMenuSelection && activeMenuItem === 'statistics'
-                  }
-                />
-              )}
-              {showLeaderboard && hasPerformanceLeaderboards && (
-                <NavigationItem
-                  icon={() => <Icon name="Medal" size="medium" />}
-                  label={intl.formatMessage(navigationMessages['leaderboards'])}
-                  onClick={goToLeaderBoardsView}
-                  id="navigation_leaderboards"
-                  isSelected={
-                    enableMenuSelection && activeMenuItem === 'leaderboards'
-                  }
-                />
-              )}
-              {hasPerformance && userDetails && (
-                <NavigationItem
-                  icon={() => <Icon name="ChartBar" size="medium" />}
-                  label={intl.formatMessage(navigationMessages['performance'])}
-                  onClick={() => props.goToPerformanceViewAction()}
-                  id="navigation_report"
-                  isSelected={
-                    enableMenuSelection &&
-                    activeMenuItem === WORKQUEUE_TABS.performance
-                  }
-                />
-              )}
-            </>
-          }
-          {hasPerformanceVitalStatisticsExports && (
-            <NavigationItem
-              icon={() => <Icon name="Export" size="medium" />}
-              id={`navigation_${WORKQUEUE_TABS.vsexports}`}
-              label={intl.formatMessage(
-                navigationMessages[WORKQUEUE_TABS.vsexports]
-              )}
-              onClick={goToVSExportsAction}
-              isSelected={
-                enableMenuSelection &&
-                activeMenuItem === WORKQUEUE_TABS.vsexports
-              }
-            />
-          )}
-        </NavigationGroup>
-      )}
+      <OrganisationGroup
+        intl={intl}
+        enableMenuSelection={enableMenuSelection}
+        activeMenuItem={activeMenuItem}
+        goToAllUserEmail={goToAllUserEmail}
+        goToOrganisationView={props.goToOrganisationViewAction}
+        goToPerformanceView={props.goToPerformanceViewAction}
+        goToSystemList={goToSystemViewAction}
+        goToTeamView={props.goToTeamViewAction}
+      />
+      <PerformanceGroup
+        intl={intl}
+        activeMenuItem={activeMenuItem}
+        enableMenuSelection={enableMenuSelection}
+        goToDashboardView={goToDashboardView}
+        goToLeaderBoardsView={goToLeaderBoardsView}
+        goToPerformanceStatistics={goToPerformanceStatistics}
+        goToPerformanceView={props.goToPerformanceViewAction}
+        goToVSExport={goToVSExportsAction}
+      />
```

Comment:
Consider using the `useIntl` hook directly within components and managing navigation inside components using hooks like `useNavigate` to reduce the number of props.